src/index.js

```diff
@@ -16,7 +17,7 @@ export * as beta from './beta';
  * @returns {Promise}
  */
 export function execute(...operations) {
-  return state => {
+  return (state) => {
```

Comment:
In other adaptors, we've been using a `.prettierrc` that looks like this:
```
{
  "trailingComma": "es5",
  "tabWidth": 2,
  "singleQuote": true,
  "arrowParens": "avoid",
  "printWidth": 80
}
```
Do you have opinions here? Best to standardise so we don't see commits with style changes lumped in with functional changes.